notebooks/rapids_scanpy_funcs.py

```diff
@@ -208,6 +208,11 @@ def filter_cells(sparse_gpu_array, min_genes, max_genes, rows_per_batch=10000, b
     barcodes : series
         cudf series containing cell barcodes.
 
+        
+    on_device : bool
```

Comment:
In situations where we can't perform this on GPU, why not just use the scanpy filter_cells function?